sagemaker-pipelines/nlp/amazon_comprehend_sagemaker_pipeline/sm_pipeline_with_comprehend.ipynb

```diff
@@ -0,0 +1,961 @@
+{
```

Comment:
<p>This direct S3 -&gt; S3 copy will only work when run from the same region that <code>s3://aws-ml-blog</code>  exists in. I recommend doing a copy from target S3 to local, then another copy from local to source S3. That will work within all commercial regions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2979/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>